docs/edge.rst

```diff
@@ -0,0 +1,253 @@
+.. _edge:
+
+============
+CrateDB Edge
+============
+
+Crate.io is pleased to `announce CrateDB Edge`_, the hybrid cloud database
+solution integrating local control over regions and data storage with the
+managed software stack from CrateDB Cloud.
+
+The process of getting CrateDB Edge running is less complicated than it may
```

Comment:
Suggestion: Phrase this positively and claim that it is "far easier" instead of "less complicated"